sql/core/src/test/resources/sql-tests/inputs/join.sql

```diff
@@ -0,0 +1,225 @@
+-- join nested table expressions (auto_join0.q)
```

Comment:
: ) That is for helping reviewers know the origins of the queries. If you think we do not care, we can remove it.